DevTools/Program.cs

```diff
@@ -6,8 +6,9 @@ static void Main(string[] args)
 		{
 //			CharacterListProcessing.Process();
 //			BiblicalTerms.Processor.Process();
-			FCBH.Processor.Process();
+//			FCBH.Processor.Process();
 //			VerseBridgeHelper.RemoveAllVerseBridges();
+            TermTranslator.Processor.Process();
```

Comment:
I think everything in Glyssen currently uses tabs.  Let's change your VS settings to tabs.